packages/gatsby-plugin-emotion/README.md

```diff
@@ -14,26 +14,47 @@ guide](https://emotion.sh/docs/migrating-to-emotion-10#incremental-migration) fo
 This plugin lets you leverage Emotion's server-side rendering (SSR) on the following Emotion APIs:
 
 - `css` React component prop from `@emotion/core` with the `jsx` babel transformation:
-```jsx
-import { css } from '@emotion/core'
 
-const Header = () => <h2 css={css`color: hotpink;`}>Welcome</h2>    // ✅ SSR
+```jsx
+import { css } from "@emotion/core"
+
+const Header = () => (
+  <h2
+    css={css`
+      color: hotpink;
+    `}
+  >
+    Welcome
+  </h2>
+) // ✅ SSR
 ```
+
 - `styled` function with `@emotion/styled`:
+
 ```jsx
-import styled from '@emotion/styled'
+import styled from "@emotion/styled"
 
-const Header = styled.h2`color: hotpink;`   // ✅ SSR
+const Header = styled.h2`
+  color: hotpink;
+` // ✅ SSR
 ```
 
 NOTE: Unlike the APIs mentioned above, the `css` function from `emotion` **will not leverage SSR using this plugin**:
-```jsx
-import { css } from 'emotion'
 
-const Header = () => <h2 className={css`color: hotpink;`}>Welcome</h2>    // ✖ SSR
+```jsx
+import { css } from "emotion"
+
+const Header = () => (
+  <h2
+    className={css`
+      color: hotpink;
+    `}
+  >
+    Welcome
+  </h2>
+) // ✖ SSR
```

Comment:
@wardpeet :
> Can you put in an example? I'm not 100% sure what you mean with this line.

I added some example snippets
Is it easier to understand now?

Reply:
Oh yeah definitely, Thank you for adding these.